src/Illuminate/Foundation/Validation/ValidatesRequests.php

```diff
@@ -28,9 +28,13 @@
      *
      * @throws \Illuminate\Foundation\Validation\ValidationException
      */
-    public function validate(Request $request, array $rules, array $messages = [], array $customAttributes = [])
+    public function validate($request, array $rules, array $messages = [], array $customAttributes = [])
```

Comment:
given this is the validate request trait, i still don't think we should allow anything other than a request in here